### ISSUE TITLE
Error with IgxAutocompleteDirective when building Angular ssr

### ISSUE THREAD

**Issue by irm4nt45** [IgniteUI/igniteui-angular#6938]
## Description  
Getting this error when trying to build Angular Universal application. 

 * igniteui-angular version: ^8.2.3
 * browser: any

## Steps to reproduce  

Getting this bug after `ng run build:ssr`:

```
ERROR in src/app/recipes/recipes.component.html(4,13): Directive IgxAutocompleteDirective, Expected 0 arguments, but got 1.
src/app/recipes/recipes.component.html(4,13): Directive IgxAutocompleteDirective, Expected 0 arguments, but got 1.
src/app/recipes/recipes.component.html(4,13): Directive IgxAutocompleteDirective, Expected 0 arguments, but got 1.
```

## Result  
Bug report

## Expected result  
While the application is working fine when serving on a browser it struggles to work on the server side.

## Attachments  
Attach a sample if available, and screenshots, if applicable.  

Here are my `package.json` dependencies:

```
"dependencies": {
    "@angular/animations": "~8.2.11",
    "@angular/cdk": "~8.2.3",
    "@angular/common": "~8.2.11",
    "@angular/compiler": "~8.2.11",
    "@angular/core": "~8.2.11",
    "@angular/fire": "^5.4.2",
    "@angular/forms": "~8.2.11",
    "@angular/material": "^8.2.3",
    "@angular/platform-browser": "~8.2.11",
    "@angular/platform-browser-dynamic": "~8.2.11",
    "@angular/platform-server": "~8.2.11",
    "@angular/router": "~8.2.11",
    "@fullcalendar/angular": "^4.4.5-beta",
    "@fullcalendar/core": "^4.4.0",
    "@fullcalendar/daygrid": "^4.4.0",
    "@fullcalendar/interaction": "^4.4.0",
    "@fullcalendar/resource-timegrid": "^4.4.0",
    "@fullcalendar/timegrid": "^4.4.0",
    "@nguniversal/express-engine": "^8.1.1",
    "@nguniversal/module-map-ngfactory-loader": "8.1.1",
    "@types/hammerjs": "^2.0.36",
    "core-js": "^3.6.4",
    "cors": "^2.8.5",
    "domino": "^2.1.4",
    "express": "^4.15.2",
    "firebase": "^7.11.0",
    "hammerjs": "^2.0.8",
    "igniteui-angular": "^8.2.3",
    "jszip": "^3.1.5",
    "minireset.css": "~0.0.4",
    "resize-observer-polyfill": "^1.5.1",
    "rxjs": "~6.4.0",
    "rxjs-compat": "^6.5.4",
    "tslib": "^1.10.0",
    "web-animations-js": "^2.3.2",
    "xmlhttprequest": "^1.8.0",
    "zone.js": "~0.9.1"
  }
```

And my code, mostly copy paste from official example:
```
<igx-input-group type="search" class="searchBox" *ngIf="isBrowser">
            <input 
            #input1 
            igxInput 
            name="recipeList" 
            placeholder="Search recipes by name" 
            [igxAutocomplete]='recipesPanel' 
            type="text"
            [(ngModel)]="recipeSelected"
            />
            <label igxLabel for="recipeList">All receipes</label>
            <igx-prefix *ngIf="input1.value.length > 0" (click)=search()>
                <igx-icon>search</igx-icon>
            </igx-prefix>
            <igx-suffix *ngIf="input1.value.length > 0" (click)="clear();" >
                <igx-icon>clear</igx-icon>
            </igx-suffix>
        </igx-input-group>

        <igx-drop-down #recipesPanel maxHeight="300px">
            <igx-drop-down-item *ngFor="let recipe of recipeList | startsWith:recipeSelected" [value]="recipe" (click)='search();'>
                {{ recipe }}
            </igx-drop-down-item>
        </igx-drop-down>
```

```
recipeListText = [];
recipeSelected;

constructor(
private router: Router, 
private recipeService: TheMealDbService, 
fb: FormBuilder, @Inject(PLATFORM_ID) private platformId
) {
    this.isBrowser = isPlatformBrowser(platformId);
    this.recipeList = 
[
      "Apple Frangipan Tart", "Apple & Blackberry Crumble", 
      "Bakewell tart"
] 
  }
search(){
  console.log('this.recipeSelected);
}
public clear() {
  console.log(this.recipeSelected);
}
```

```
@Pipe({ name: "startsWith" })
export class AutocompletePipeStartsWith implements PipeTransform {
    public transform(collection: any[], term = "") {
        return collection.filter((item) => item.toString().toLowerCase().startsWith(term.toString().toLowerCase()));
    }
```

**Comment by Lipata** [comment]
@irm4nt45, I think we fixed that in version [8.2.7](https://github.com/IgniteUI/igniteui-angular/releases/tag/8.2.7), with bug #6057. [Here](https://github.com/IgniteUI/igniteui-angular/commit/37267e98ccd41bf7af62557f7c2a9fdf15e4a00f#diff-1abfeef9c488893630f5609e988d9f1a) is the fix itself. Can you upgrade to that version?

**Comment by irm4nt45** [comment]
igniteui-angular update to version 8.2.7 fixed the issue.
